README.md

```diff
@@ -1,13 +1,18 @@
 # AirConsole Construct2
-Construct2 Plugin for AirConsole Version 1.3
-
+Construct2 Plugin for AirConsole Version 1.5.0
 ## Installation
 Clone this repo or [download files as zip](https://github.com/AirConsole/airconsole-construct2/archive/master.zip) and drag and drop the airconsole.c2addon file into the Construct2 application.
 
 More information about the plugin:
 
 http://developers.airconsole.com/#/guides/construct2
 
+## Changes in version 1.4.2
+* Added ControllerDeviceIDs. See (http://puu.sh/s0wNv.png) for usage
```

Comment:
We can make it 1.4.1 for the whole pull request
I like the example picture! Gonna include it in the documentation as soon as its merged!